src/ContosoTraders.Ui.Website/src/pages/detail/detail.scss

```diff
@@ -1,6 +1,8 @@
 // -----------------------------------------------------------------------------
 // This file contains styles that are specific to the Detail Products page.
 // -----------------------------------------------------------------------------
+@import '../../styles/abstracts/variables';
```

Comment:
same here. can we not include variables in one place only? I would normally use css variables and have a fall back so if detail page could render outside the app and still work but inside the app it would have access to the variables for styling etc